src/NServiceBus.Core/Configure_Obsolete.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace NServiceBus
+{
+    using NServiceBus.Config;
+
+    public partial class Configure
+    {
+        [ObsoleteEx(Message = "Remove when IWantToRunWhenConfigurationIsComplete is obsoleted.", RemoveInVersion = "7.0")]
+        void RegisterWantToRunWhenConfigurationIsCompleteAsInstancePerCall()
```

Comment:
I introduced them as a reminder to remove in next version. Or should I
remove the code and no longer execute them?